#!/usr/bin/env python
PACKAGE = "mastermind"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("imu_gain", double_t, 0, "A double parameter", 1, 0,   1e8)

exit(gen.generate(PACKAGE, "imu_cov", "imu_covariance"))
